docs/yeetsheet-doc.md

```diff
@@ -0,0 +1,205 @@
+
+# YeetSheet Documentation
+
+## Table of Contents
+1. [Project Overview](#project-overview)
+2. [Installation](#installation)
+3. [Usage](#usage)
+4. [Running the Project](#running-the-project)
+5. [Features](#features)
+6. [Configuration](#configuration)
+7. [How to Contribute](#how-to-contribute)
+8. [Development Setup](#development-setup)
+
+## Project Overview
+
+### Project Name
+**YeetSheet**
+
+**Description**: A lightweight spreadsheet web application that can be used in any browser to manage tabular data with JavaScript syntax. Designed for ease of use and portability, meaning it can be used either online or offline. YeetSheet is perfect for developers, educators, students, and businesses.
+
+### Project Links
+- [GitHub (main)](https://github.com/NoroffFEU/yeetsheet)
+- [GitHub (workflow)](https://github.com/NoroffFEU/yeetsheet/tree/workflow)
+- [README.md](https://github.com/NoroffFEU/yeetsheet/blob/main/README.md)
+- [Netlify](https://yeetsheet.netlify.app/)
+
+### Purpose
+This project's purpose is to create a lightweight browser-based web application for managing tabular data using JavaScript syntax instead of the proprietary language other traditional spreadsheet applications use.
+
+#### Some problems this project aims to solve are:
+1. **Complex Syntax and Legacy Issues**:
+    - **Legacy Constraints**: The syntax of many existing spreadsheet applications, although powerful, has accumulated numerous legacy constraints over the decades since their inception.
+    - **Steep Learning Curve**: Functions and syntax in many traditional spreadsheets are often not intuitive, posing a significant challenge for new users to master.
+
+2. **Portability and Accessibility**:
+    - **Device Independence**: Unlike traditional spreadsheets that often require specific software installations or subscriptions, YeetSheet is entirely portable and accessible on any device with a browser.
+    - **Code and Data Sharing**: Traditional CSV files are limited in preserving functions and often need mainstream applications for optimal viewing. YeetSheet overcomes this by allowing seamless code and data sharing without these restrictions.
+
+### Technologies
+**Main Technologies**:
+- JavaScript
+- HTML
+- CSS
+
+**Libraries and Frameworks**:
+- Tailwind CSS
+- Vite
+- Cypress
+- Husky
+- Prettier
+- ESLint
+- Babel
+- Jest
```

Comment:
Hi, we don't use jest, but we swicted to vitest after the QA team learnt that vitest is the official testing suite framework, if I'm not wrong. Jest as dependency is still there because has not been removed.